lib/color_const.dart

```diff
@@ -6,5 +6,6 @@ class KColors {
   static const Color basedYellowColor = Color(0xffffcf00);
   static const Color basedOrangeColor = Color(0xffee6123);
   static const Color basedRedColor = Color(0xfffa003f);
+  static const Color basedBlueColor = Color(0xff42bff5);
```

Comment:
What is meaning of 'based'?

Reply:
done

Reply:
done? What do you mean by done? I've asked you a question